commit 86be71490e02d2c3ba933861fb89376b8f2ca92d
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:21:44 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-no-CCPRINT.diff

diff --git a/src/port1.0/portmain.tcl b/src/port1.0/portmain.tcl
index 2a86e49d3..dae406f74 100644
--- a/src/port1.0/portmain.tcl
+++ b/src/port1.0/portmain.tcl
@@ -157,7 +157,7 @@ default universal_possible {[expr {${os.universal_supported} && [llength ${confi
 
 default compiler.cpath {${prefix}/include}
 default compiler.library_path {${prefix}/lib}
-default compiler.log_verbose_output yes
+default compiler.log_verbose_output no
 default compiler.limit_flags no
 default compiler.support_environment_paths no
 default compiler.support_environment_sdkroot no
